llvm/test/tools/llvm-no-spir-kernel/has-spir-kernel.ll

```diff
@@ -1,9 +1,8 @@
-; RUN: not llvm-no-spir-kernel  %s 
```

Comment:
This test checks the same function as has-spir-kernel2.ll
I suppose that we may simply remove this test as duplicate.

Reply:
Done